packages/fa-icon-chooser/src/components/fa-icon-chooser/fa-icon-chooser.tsx

```diff
@@ -449,7 +455,15 @@ export class FaIconChooser {
   }
 
   isSharpSolidAvailable() {
-    return this.pro() && !!this.resolvedVersion().match('(6.2[0-9]*.)');
+    return this.pro() && !!this.resolvedVersion().match('(6.4[0-9]*.)');
```

Comment:
I think this would fail for FA 6.3.0 and 6.2.1.

This was a bug Frances and I previously found when we tracked down why the chooser wasn't working for Sharp Solid in FA 6.3.0.

We (I?) should probably not have used regular expressions here, because it seems all to easy make such a mistake based on literal string matching.

Looks like we've already got the `semver` NPM in this bundle, so it seems better to do something like this for Sharp Solid:

```javascript
semver.satisfies(this.resolvedVersion(), '>=6.2.0')
```

Each of the Sharp styles have _different_ minimum versions, though. This is correct for Sharp Solid, but not Sharp Regular nor Sharp Light.